.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: generic
 dist: trusty
-addons:
-  apt:
-    packages:
-      - emacs
+env:
+  global:
+    - CURL="curl -fsSkL --retry 9 --retry-delay 9"
+  matrix:
+    - EMACS_VERSION=24.3
+    - EMACS_VERSION=25
+install:
+  - $CURL -O https://github.com/npostavs/emacs-travis/releases/download/bins/emacs-bin-${EMACS_VERSION}.tar.gz
+  - tar xf emacs-bin-${EMACS_VERSION}.tar.gz -C /
+  - export EMACS=/tmp/emacs/bin/emacs
+  - $EMACS --version
 script: make test
+before_deploy: cd doc && make doc
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  local_dir: public
+  on:
+    branch: doc-site-cd
```

Comment:
@jwiegley To your question about the context for specifying the branches, @nickmccurdy means this line.

Reply:
Can you make a PR with the changes that are missing? I don't understand the point being made yet...